static/resources/json/datadog_collection.json

```diff
@@ -3184,7 +3184,7 @@
 									"raw": "{\n      \"scope\": \"host:i-123\"\n}"
 								},
 								"url": {
-									"raw": "https://api.datadoghq.com/api/v1/downtime/cancel/by_scope?api_key={{dd_api_key}}&application_key={{dd_app_key}}",
+									"raw": "https://app.datadoghq.com/api/v1/downtime/cancel/by_scope?api_key={{dd_api_key}}&application_key={{dd_app_key}}",
```

Comment:
`api`?

Reply:
that's weird.. i know but when looking at the postman collection the values are correct